ivy_tests/test_ivy/helpers.py

```diff
@@ -328,7 +328,9 @@ def trim(*, docstring):
     return "\n".join(trimmed)
 
 
-def docstring_examples_run(*, fn, from_container=False, from_array=False):
+def docstring_examples_run(
```

Comment:
Please revert all changes made to this file as it's not related to the function you're working on